### DISCUSSION TITLE
I got errors with spacy-llm==0.5.1

### DISCUSSION THREAD

**Discussion by innocent-charles** [explosion/spacy-llm#303] [Q&A]
I have upgraded the spacy-llm version to 0.5.1 and used the task of spacy.NER.v3 and model of spacy.GPT-3-5.v2. 

But unfortunately I got the following error , what to do ? 

```python 
File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for NERCoTExample
spans
  field required (type=value_error.missing)```

**Comment by innocent-charles** [comment]
Any help please !

**Comment by rmitsch** [comment]
Continuing the discussion in https://github.com/explosion/spacy-llm/issues/304.
